.github/workflows/node.js.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deployment to AWS lambda
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  serverless-deploy:
```

Comment:
Maybe we can make this more generic, like `build`? Since you're also setting up, testing and then deploying. Just a thought

Reply:
Ye i agree but i wasn't sure if we were going to use the same Action CI for testing. If yes then for sure. I think a separate action could be better, to spin up a separate VM for testing because of memory restriction. Needs investigation tbh

Reply:
Keep in mind the goal of the PR was only for the deployment of the functions. 
We should make a separate PR and issue for CI testing.

Reply:
1) I would be careful with more actions because there is a limit on the free-tier if I understand correctly. Usually you don't need a separate job for running tests and deploying but I see no problem with separating them for our case.

2) Agreed, we can add this as a separate issue!

Reply:
Good call yea, yea i think the goal would be in one